### PR TITLE
Remove YAML from codebase

### LINKED ISSUE #218: Nuke YAML as a format
YAML is shit, lets nuke it.

### PR DESCRIPTION
**Pull request by TheNeikos**
Closes #218

### CODE REVIEW

**Comment by TheNeikos** [comment]
It didn't feel right to have the changelog as its own commit here? But I can split it out if you'd rather have that

File: .changelog.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ edit_data = true
 #
 # Right now, only "yaml" is supported
```
Comment (matthiasbeyer) [inline .changelog.toml:29]: This is not true anymore, is it?

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
Can you also convert the existing changelogs, so that the tool can read its own historical changelogs?

File: .changelogs/unreleased/2023-03-03T08_08_41_21078057.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
----
-subject: '"Show" subcommand'
-type: Feature
++++
+subject = "'\"Show\" subcommand'"
```
Comment (matthiasbeyer) [inline .changelogs/unreleased/2023-03-03T08_08_41_21078057.md:2]:
```suggestion
subject = "\"Show\" subcommand"
```

Comment (matthiasbeyer) [reply .changelogs/unreleased/2023-03-03T08_08_41_21078057.md:2]: Just a nit, and not really important IMO.

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
bors merge

**Pull request merged**